docs/src/test/scala/docs/http/scaladsl/ServerShutdownExampleSpec.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2009-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.http.scaladsl
+
+import akka.Done
+import akka.actor.{ ClassicActorSystemProvider, CoordinatedShutdown }
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.Behaviors
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.Http.ServerBinding
+import akka.http.scaladsl.server.Directives._
+import docs.CompileOnlySpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import scala.concurrent.duration._
+import scala.concurrent._
+import scala.io.StdIn
+
+class ServerShutdownExampleSpec extends AnyWordSpec with Matchers
+  with CompileOnlySpec {
+
+  // #suggested
+  def addToCoordinatedShutdown(terminationGracePeriod: FiniteDuration)(binding: ServerBinding)(implicit system: ClassicActorSystemProvider): ServerBinding = {
```

Comment:
Wouldn't people copy and paste this exact snapshot everywhere? Could we either support it as a configuration option or at least just provide that exact method somewhere in the Http extension?